lib/stdlib/doc/src/binary.xml

```diff
@@ -575,6 +575,17 @@ store(Binary, GBSet) ->
       </desc>
     </func>
 
+    <func>
+      <name name="secure_compare" arity="2" since="OTP R24"/>
+      <fsummary>Compares two binaries in constant time.</fsummary>
+      <desc>
+          <p>Given two binaries this function will iterate and compare each byte of both binaries in constant time 
```

Comment:
Docs could be improved here. Specifically some use case examples perhaps and when not to use this function. Possibly also link to https://en.wikipedia.org/wiki/Timing_attack